### ISSUE TITLE
[Feature Request] Please provide a CRX download on the website or on GitHub

### ISSUE THREAD

**Issue by cookiengineer** [darkreader/darkreader#3965]
Hey,

I wanted to first say that I've been using Darkreader a lot in the past, and even on my mobile devices in Firefox and it worked like a charm. However, since I changed my OS setup and switched to Ungoogled-Chromium as a Browser, I am not able to use your extension anymore without having to rebuild it manually every time a release appears.

Therefore I am asking kindly if there would be a possibility to automatically upload the CRX file into either the GitHub releases section or to provide an alternative download for it on the website?

The underlying reason is that the "grabbing" mechanism has changed since around a month ago, so you're not able to download the extension via the `clients2.google.com/service/update2/crx?...` trick anymore, and therefore users like me that care about Privacy would need an alternate source for the CRX file in order to use this extension.

Thanks in advice for consideration,
~Chris

**Comment by Gusted** [comment]
Hi @cookiengineer!

I'm all in for an alternative option to upload the .crx and .xpi on the release tab for the [current tags](https://github.com/darkreader/darkreader/tags) I will discuss this with @alexanderby.

Regards,
Gusted

**Comment by hazarek** [comment]
@cookiengineer I found this, it works  https://github.com/NeverDecaf/chromium-web-store

**Comment by cookiengineer** [comment]
@hazarek Yes, I know this extension. But my previous workflow was to just download the CRX and install them locally via developer mode in (ungoogled) chromium.

I tried to verify that it works, but it doesn't in my case. I also tried to rebuild the URL the way they do [1], but I always get a 0 byte file as a response.

[1] https://github.com/NeverDecaf/chromium-web-store/blob/c2336c8c4182e18ee6022d1f740c68dad9742c98/src/scripts/util.js#L49-L64

**Comment by hazarek** [comment]
@cookiengineer https://github.com/dashokkumar93/crx-download
